src/cargo/util/command_prelude.rs

```diff
@@ -281,7 +281,9 @@ pub fn multi_opt(name: &'static str, value_name: &'static str, help: &'static st
 }
 
 pub fn subcommand(name: &'static str) -> App {
-    App::new(name).setting(AppSettings::DeriveDisplayOrder | AppSettings::DontCollapseArgsInUsage)
+    App::new(name)
+        .dont_collapse_args_in_usage(true)
```

Comment:
FYI this became a `global_setting`.

If that is a concern, you can still use the `AppSetting` until clap v4.  In clap v4, my expectation is we'll support subcommands overriding global settings (I think that doesn't work today)

When we made this global, it was under the assumption people would want a uniform look.  If this assumption is incorrect, we can open an issue and re-visit this in clap v4.

Reply:
Huh, interesting. I honestly couldn't tell you whether that change is okay or not. I feel like it _probably_ is since we're seemingly applying it to all subcommands, but others who know the CLI interface better may need to weigh in here.